environment/src/main/java/com/sequenceiq/environment/environment/service/freeipa/FreeIpaInstanceCountByGroupProvider.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.environment.environment.service.freeipa;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.environment.api.v1.environment.model.request.AttachedFreeIpaRequest;
+
+@Service
+public class FreeIpaInstanceCountByGroupProvider {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FreeIpaInstanceCountByGroupProvider.class);
+
+    @Value("${environment.freeipa.groupInstanceCount.default}")
```

Comment:
shouldn't there be a default value?

Reply:
There was discussions about having a default when it is already presents on the `application.yml` level, I can only remember that somebody recommended to use default from the `yml` and not having default value in the spring expression is more straight forward.
But I haven't seen this approach as a must have :)

Reply:
no. :) as we discussed.